clustercontroller-core/src/test/java/com/yahoo/vespa/clustercontroller/core/DatabaseTest.java

```diff
@@ -154,7 +154,7 @@ private void setWantedState(Node n, NodeState ns, Map<Node, NodeState> wantedSta
         Request req = new Request("setNodeState");
         req.parameters().add(new StringValue("storage/cluster.mycluster/" + n.getType().toString() + "/" + n.getIndex()));
         req.parameters().add(new StringValue(ns.serialize(true)));
-        connection.invokeSync(req, timeoutS);
+        connection.invokeSync(req, timeout.getSeconds());
```

Comment:
getSeconds will round down. 3.5 => 3, 0.9 => 0.
That might be fine here in the test, but generally it is not. Can Duration be converted to a double in seconds ?

Reply:
Good point, will fix